packages/js-api/src/core/frigade.ts

```diff
@@ -341,20 +378,26 @@ export class Frigade extends Fetchable {
               } as FlowStateDTO),
             })
 
-        const rulesData = new Map()
+        const rulesData: RulesList = new Map()
```

Comment:
Here and everywhere else -- should we rename everywhere it says s/rule/collection in file names and data structures? Basically I've bene trying to remove rules everywhere except for our internal graphql API which I will migrate too.

Reply:
Yeah, agree. There's a whole pass to do of renaming all the internals. Will throw that in a followup PR today.